packages/near-api-js/.gitignore

```diff
@@ -0,0 +1,21 @@
+#IDE
+.idea
+
+storage
+node_modules
+*.map
+
+dist/
+
+# near-api-js files & coverage
+neardev/
+coverage/
+.nyc_output
+src\.ts/protos\.d\.ts
+package\-lock\.json
+
+# OS X
+.DS_Store
+
+# Generated typedoc files
+docs/
```

Comment:
I can see we still have the root `.gitignore`, is this even needed here? there's nothing specific to `near-api-js`.